<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nomad Resource Explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="m_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_tabShips">
       <attribute name="title">
        <string>Ships</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" columnstretch="4,3">
        <item row="1" column="1">
         <widget class="QTableWidget" name="m_shipInventoryTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Inventory:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QTableWidget" name="m_shipTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Class</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pilot</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Location</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_tabPlaces">
       <attribute name="title">
        <string>Places</string>
       </attribute>
       <layout class="QGridLayout" name="placeTabLayout" columnstretch="1,2">
        <item row="0" column="1">
         <widget class="QGraphicsView" name="m_planetView"/>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="m_placeLabelGrid">
          <item row="4" column="0">
           <widget class="QLabel" name="m_placeRepLabel">
            <property name="text">
             <string>Planet representative:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="m_placeTypeLabel">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="m_placeTypeData">
            <property name="text">
             <string>&lt;type&gt;</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="m_placeRepData">
            <property name="text">
             <string>&lt;rep&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="m_placeClassData">
            <property name="text">
             <string>&lt;class&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_placeTemperatureLabel">
            <property name="text">
             <string>Temperature:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="m_placeRaceData">
            <property name="text">
             <string>&lt;race&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_placeRaceLabel">
            <property name="text">
             <string>Alien race:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_placeClassLabel">
            <property name="text">
             <string>Class:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="m_placeTemperatureData">
            <property name="text">
             <string>&lt;temp&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QTableWidget" name="m_placeTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_tabAliens">
       <attribute name="title">
        <string>Aliens</string>
       </attribute>
       <layout class="QGridLayout" name="m_alienTabLayout" columnstretch="3,5">
        <item row="2" column="1">
         <layout class="QGridLayout" name="m_alienDataGrid">
          <item row="0" column="0" colspan="2">
           <widget class="QSlider" name="m_alienFrameSlider">
            <property name="maximum">
             <number>63</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QGraphicsView" name="m_alienView"/>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QTableWidget" name="m_alienTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Race</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="m_alienAnimationFrameLabel">
          <property name="text">
           <string>Animation frames:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_tabObjects">
       <attribute name="title">
        <string>Objects</string>
       </attribute>
       <layout class="QGridLayout" name="m_objectTabLayout" columnstretch="2,1">
        <item row="3" column="1">
         <widget class="QLabel" name="m_objectTypeLabel">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGraphicsView" name="m_objectImageView"/>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="m_objectHLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="m_objectUniqueLabel">
          <property name="text">
           <string>Unique:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="5">
         <widget class="QTableWidget" name="m_objTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>14</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Altec Hocker</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Arden</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Bellicosian</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Chanticleer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Human</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kenelm</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Korok</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Musin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pahrump</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phelonese</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shaasa</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ursor</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="m_objDescriptionLabel">
          <property name="text">
           <string>Inventory objects with relative values to each alien race:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_tabFacts">
       <attribute name="title">
        <string>Facts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="m_factIdsLabel">
          <property name="text">
           <string>Fact IDs:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="4">
         <widget class="QListWidget" name="m_factList">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="m_factText"/>
        </item>
        <item row="2" column="1">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="m_factReceptivityLabel">
          <property name="text">
           <string>Receptivity of each race to this fact:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QGridLayout" name="m_factBarGrid">
          <item row="4" column="1">
           <widget class="QProgressBar" name="m_factBarHuman">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="m_factLabelKenelm">
            <property name="text">
             <string>Kenelm:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="m_factLabelMusin">
            <property name="text">
             <string>Musin:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="m_factLabelUrsor">
            <property name="text">
             <string>Ursor:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QProgressBar" name="m_factBarShaasa">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="m_factLabelHuman">
            <property name="text">
             <string>Human:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QProgressBar" name="m_factBarKorok">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="m_factBarArden">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QProgressBar" name="m_factBarKenelm">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="m_factLabelPhelonese">
            <property name="text">
             <string>Phelonese:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_factLabelArden">
            <property name="text">
             <string>Arden:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QProgressBar" name="m_factBarMusin">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QProgressBar" name="m_factBarChanticleer">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QProgressBar" name="m_factBarUrsor">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QProgressBar" name="m_factBarBellicosian">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="m_factLabelKorok">
            <property name="text">
             <string>Korok:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="m_factLabelPahrump">
            <property name="text">
             <string>Pahrump:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QProgressBar" name="m_factBarPhelonese">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="m_factLabelAltec">
            <property name="text">
             <string>Altec Hocker:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QProgressBar" name="m_factBarAltec">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_factLabelChanticleer">
            <property name="text">
             <string>Chanticleer:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_factLabelBellicosian">
            <property name="text">
             <string>Bellicosian:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QProgressBar" name="m_factBarPahrump">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="m_factLabelShaasa">
            <property name="text">
             <string>Shaasa:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="m_factHSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_tabSounds">
       <attribute name="title">
        <string>Sounds</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0,0,0">
        <item row="0" column="0" rowspan="4">
         <widget class="QTreeWidget" name="m_soundTree">
          <column>
           <property name="text">
            <string notr="true">Sound containers</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="m_soundPlayButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="m_soundNextButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="m_soundStopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="m_soundPrevButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="m_soundStateLabel">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string>Stopped</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLabel" name="m_soundIDLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sound ID:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_game_data_dir"/>
    <addaction name="actionOpen_saved_game"/>
    <addaction name="actionClose_data_files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_game_data_dir">
   <property name="text">
    <string>Open game data directory...</string>
   </property>
  </action>
  <action name="actionOpen_saved_game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open saved game...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose_data_files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close data files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
